src/webui/www/private/views/preferences.html

```diff
@@ -1280,6 +1280,18 @@
                     <input type="text" id="requestQueueSize" style="width: 15em;" />
                 </td>
             </tr>
+            <tr>
+                <td>
+                    <label for="storageType">QBT_TR(Storage Type (requires restart):)QBT_TR[CONTEXT=OptionsDialog]&nbsp;<a href="https://www.libtorrent.org/reference-Custom_Storage.html" target="_blank">(?)</a></label>
```

Comment:
```suggestion
                    <label for="storageType">QBT_TR(Storage Type (libtorrent >= 2.0; requires restart):)QBT_TR[CONTEXT=OptionsDialog]&nbsp;<a href="https://www.libtorrent.org/reference-Custom_Storage.html" target="_blank">(?)</a></label>
```

Reply:
Also I don't think the given link points to appropriate/useful for users info.

Reply:
I don't have other good reference for this storage type, since it's on the advanced page, it might be
enought to let user know the difference between those two disk io type.